test/quantization/test_numeric_suite_fx.py

```diff
@@ -135,6 +136,34 @@ def compare_and_validate_results(float_model, q_model):
 
         compare_and_validate_results(backup_prepared_model, q_model)
 
+    @override_qengines
+    def test_compare_weights_lstm_dynamic_fx(self):
+        r"""Compare the weights of float and dynamic quantized lstm layer"""
+
+        def compare_and_validate_results(float_model, q_model):
+            weight_dict = compare_weights_fx(
+                float_model.state_dict(), q_model.state_dict()
+            )
+            self.assertEqual(len(weight_dict), 1)
```

Comment:
Could you add the check if the key in weight_dict is expected?

Reply:
Added in compare_model_stub PR